test/regress/regress0/fp/ext-rew-test.smt2

```diff
@@ -1,3 +1,4 @@
+; REQUIRES: symfpu
```

Comment:
This regression now requires symfpu to be solved?

Reply:
It did in an intermediate commit.  Regardless, this should be enabled for this benchmark since its FP.